.ci.yaml

```diff
@@ -26,7 +26,7 @@ platform_properties:
       dependencies: >-
         [
           {"dependency": "android_sdk", "version": "version:33v6"},
-          {"dependency": "open_jdk", "version": "version:11"},
```

Comment:
Non blocking but I would prefer if we jumped from 11 to 17 to align with the usages in android studio. It is way more unlikely that customers are using java 12.

Reply:
FYI, there seems to be version 17 in cipd as well: https://chrome-infra-packages.appspot.com/p/flutter/java/openjdk/linux-amd64/+/3HlEfp1nbzGbtJ4K3UQ1BM4nHieyCNc0enqBMsSRN7IC

Reply:
I've changed it to 17, and I'll see how the run goes. In the likely event that it's not just that simple, I'll file an issue with details, drop back to 12 to land this, and then revisit 17 afterward (quickly if it's not going to be a major undertaking).

Reply:
Looks like other than build-all (which we knew would fail because the legacy project has a really old AGP) everything was fine, so I'll switch to 17 instead of 12, and just keep build-all on 11.